x/concentrated-liquidity/query.go

```diff
@@ -336,3 +336,93 @@ func (k Keeper) GetNumNextInitializedTicks(ctx sdk.Context, poolId, numberOfNext
 
 	return liquidityDepths, nil
 }
+
+// GetNumNextInitializedTicks is a method that returns underlying assets of a tick range
+// Errors:
+// * ticks given not found
+// * There exists at least 1 tick inside the given range
```

Comment:
> Sr guys for the late reply, I'm keeping the current logic limited to 2 consecutive ticks because it's quite confusing with the tick range overlaps different positions.
> Lets say I have an example below, range [10, 13] is not the intersection of the 3 given positions while [10, 11] is the intersection of positions 10 and 50 have liquidity 60, [11, 13] is the intersection of 3 positions have liquidity 160. So what liquidity of range [10, 13] should be, I'm leaning towards 60 because the tick range is wider but I'm not sure.
> If u can help me with this issue it would be great thank so so much
> cc @pysel @mattverse
> <img alt="Ảnh chụp Màn hình 2023-09-25 lúc 21 41 18" width="775" src="https://user-images.githubusercontent.com/72878483/273539713-5c52ace4-27ad-4037-a910-73ff8a4ac864.png">